stablehlo/reference/Interpreter.cpp

```diff
@@ -93,6 +93,16 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       Tensor runtimeOpr = fetchOperand(floorOp.operand());
       Tensor runtimeResult = eval(floorOp, runtimeOpr);
       populateResults({runtimeResult});
+    } else if (auto maxOp = dyn_cast<MaxOp>(op)) {
+      Tensor runtimeLhs = fetchOperand(maxOp.lhs());
+      Tensor runtimeRhs = fetchOperand(maxOp.rhs());
+      Tensor runtimeResult = eval(maxOp, runtimeLhs, runtimeRhs);
+      populateResults({runtimeResult});
+    } else if (auto minOp = dyn_cast<MinOp>(op)) {
+      Tensor runtimeLhs = fetchOperand(minOp.lhs());
```

Comment:
All instances of `lhs` and `rhs` will need to be updated to `getLhs` and `getRhs` before submit now that #203 is submitted

Reply:
Thanks for the heads-up @GleasonK